File: lecshop_api_modules/lecshop_web_admin/src/main/webapp/res/pagejs/wxsetting.js
// 基本路径
var basePath = $("#basePath").val();

// 初始化列表
var initData = function () {
    LSFetch({
        url: basePath + '/querywxsetting.htm',
        success: function (data) {
            $("input[name='id']").val(data.id);
            $("input[name='appId']").val(data.appId);
            $("input[name='appSecret']").val(data.appSecret);
            $("input[name='callBackUrl']").val(data.callBackUrl);
        }
    });
};

// 初始化列表
initData();

// 刷新列表
var refreshData = function () {
    initData();
};

//编辑微信设置
function saveSetting() {
    if (!($("#wxSettingForm").valid())) {
        return;
    }
    var wxSetting = {
        id: $("input[name='id']").val(),
        appId: $("input[name='appId']").val(),
        appSecret: $("input[name='appSecret']").val(),
        callBackUrl: $("input[name='callBackUrl']").val()
    };
    LSFetch({
        url: basePath + '/editwxsetting.htm',
        data: JSON.stringify(wxSetting),
        contentType: "application/json;charset=utf-8",
        success: function (data) {
            if (data == 1) {
                refreshData();
                $('#success_dialog').modal('show');
            }
        }
    });
}

File: lecshop_base_modules/lecshop_service/src/main/java/com/lecshop/total/wxsetting/bean/WxSetting.java
package com.lecshop.total.wxsetting.bean;

import lombok.Data;

/**
 * Created by dujinkai on 17/7/24.
 * 微信设置实体
 */
@Data
public class WxSetting {

    /**
     * 主键id
     */
    private long id;

    /**
     * 微信的appid
     */
    private String appId;

    /**
     * 微信的appSecret
     */
    private String appSecret;

    /**
     * 登录的时候 微信认证回调的地址
     */
    private String callBackUrl;

}
